execution/scan/literal_selector.go

```diff
@@ -39,8 +40,21 @@ func NewNumberLiteralSelector(pool *model.VectorPool, opts *query.Options, val f
 		currentStep: opts.Start.UnixMilli(),
 		val:         val,
 	}
+
+	// op.t = &model.NoopTimingInformation{}
```

Comment:
Any particular reason to comment this out? 🙂

Reply:
Because theAnalyze() methdod only returns Timing information struct , so I wasn't sure how to return 't' from `numberLiteralSelector`

Reply:
Hmm, but that isn't fixed. Our suggestions are there to just give you a structure of what it could look like, you can modify the types to suit

Reply:
I think we can first check if the o.t field is of type *model.TimingInformation using a type assertion and then return it

Reply:
Try pushing a commit with then 🙂